psst-gui/src/webapi/client.rs

```diff
@@ -1100,50 +1098,52 @@ impl WebApi {
         }
         let token = self.access_token()?;
 
-        let request = &RequestBuilder::new(
-            format!("http://{}/{}", "ip-api.com", "json"),
-            Method::Get,
-            None,
-        )
-        .query("fields", "260")
-        .header("Authorization", format!("Bearer {token}"));
-
-        let result: Cached<User> = self.load_cached(request, "User_info", "usrinfo")?;
```

Comment:
Why are we no longer using Cached<User>?

Reply:
Hmm, I think we ought to cache it, otherwise there will be latency in hitting `ip-api.com` everytime we load the home section.